lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
I'm not arguing against generating JavaScript, just trying to limit our footprint on `window`. I guess I'm not clear what your objection to using `Cable` is. Do you see it being more problematic?